packages/dcos-checks/extra/dcos-checks.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=DC/OS Poststart Checks: Run node checks
+[Service]
+EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=/opt/mesosphere/etc/dcos-diagnostics.env
+EnvironmentFile=-/opt/mesosphere/etc/dcos-diagnostics-extra.env
+Restart=on-failure
```

Comment:
This service shouldn't restart if it fails. It should just stay failed until the timer reactivates it.